Tests/Unit/Account/AccountTests.swift

```diff
@@ -14,8 +14,9 @@ class AccountTests: XCTestCase {
         let balanceUpdater = try Self.balanceUpdater(fogService: testFogService)
 
         let expect = expectation(description: "Checking account balance")
-        balanceUpdater.updateBalance {
-            guard let balance = $0.successOrFulfill(expectation: expect) else { return }
+        balanceUpdater.updateBalances {
```

Comment:
I feel like we should something like what's done for the http / grpc protocols... where we iterate over the supported token IDs for each test.

wdyt?

Reply:
Right now and in the near future there will only ever be MOB and MOBUSD so I think its ok to copy paste or be explicit, but in the future, when we have more tokens, and we have a way to fetch that list, it will make sense to be more dynamic and do some looping like you've suggested.

Reply:
verified this is not a performance bottleneck